### PR TITLE
chore: run more formulae checks in the online check

### PR DESCRIPTION
**Pull request by SMillerDev**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
Most days this doesn't catch anything anymore so we can try some more formulae at a time.

### CODE REVIEW

**Comment by SMillerDev** [comment]
Actually might want to continue on error too.

**Review by carlocab — COMMENTED** [review]
You should probably also disable `fail_fast` in your matrix so it doesn't quit at the first failure.

**Comment by SMillerDev** [comment]
> You should probably also disable fail_fast in your matrix so it doesn't quit at the first failure.

Yeah, that's what I meant with my comment. I'm a bit split on it though. 
One issue a day is easy to fix and allowing it to continue will increase it to max 50 issues a day, which is much more overwhelming.

**Comment by iMichka** [comment]
I'm fine with removing the fail-fast. At least you see how many are broken.
If it's too much in the long run just add it back.

File: .github/workflows/scheduled.yml [hunk 1]
```diff
@@ -45,6 +45,7 @@ jobs:
     container:
       image: ghcr.io/homebrew/ubuntu22.04:master
     needs: create_matrix
+    continue-on-error: true
```
Comment (carlocab) [inline .github/workflows/scheduled.yml:48]: This just means your jobs will always succeed regardless of the exit code.

Comment (SMillerDev) [reply .github/workflows/scheduled.yml:48]: Should it be on a step level then?

Comment (carlocab) [reply .github/workflows/scheduled.yml:48]: No, it should be `fail-fast: false` under `strategy:`

Comment (carlocab) [reply .github/workflows/scheduled.yml:48]: You can also set it at a step level, but that step will show up as 🟢 regardless of what happens there

**Review by carlocab — APPROVED** [review]

**Pull request merged**